examples/basic/basic.py

```diff
@@ -44,6 +44,18 @@ async def add(
     await ctx.respond(f"{first_number} + {second_number} = {first_number+second_number}")
 
 
+@bot.include
+@crescent.user_command
+async def my_user_command(ctx: crescent.Context, user: hikari.User | hikari.Member):
```

Comment:
`|` syntax only works with `from __future__ import annotations`. Should change this to Union, or import annotations at the top of this file.